### PR TITLE
[WWST-4808, WWST-4816, ICP-10286] Zigbee Switch: New Orvibo fingerprints and fix icons

### PR DESCRIPTION
**Pull request by FenMei007S**

### CODE REVIEW

**Comment by FenMei007S** [comment]
@tpmanley plz review it, to avoid file conflict, three tickets modifcation in a PR.
[WWST-4808]zigbee switch: add fingerprint for orvibo 1 gang switch
[WWST-4816]zigbee outlet
[ICP-10321] device icon incorrectm should change device type to plug

**Comment by tpmanley** [comment]
@FenMei007S  ICP-10321 doesn't appear to be the correct ticket. Can you double check the number?

File: devicetypes/smartthings/zigbee-switch.src/i18n/messages.properties [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+
+# Chinese
+'''Orvibo Smart Switch'''.zh-cn=智能墙面开关(一路版)
+'''Orivbo Smart Outlet'''.zh-cn=智能插座
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-switch.src/i18n/messages.properties]: Should be `Orvibo`

Comment (FenMei007S) [reply devicetypes/smartthings/zigbee-switch.src/i18n/messages.properties]: https://smartthings.atlassian.net/browse/ICP-10321

Comment (tpmanley) [reply devicetypes/smartthings/zigbee-switch.src/i18n/messages.properties]: @FenMei007S please double check that ticket. That one doesn't appear to have anything to do with this device.

**Review by greens — APPROVED** [review]

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 2]
```diff
@@ -33,9 +33,11 @@ metadata {
...
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0006, 0402, 0B05, FC01, FC02", outClusters: "0003, 0019", manufacturer: "iMagic by GreatStar", model: "1113-S", deviceJoinName: "Iris Smart Plug", ocfDeviceType: "oic.d.smartplug"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "eWeLink", model: "SA-003-Zigbee", deviceJoinName: "eWeLink SmartPlug (SA-003)"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "SONOFF", model: "BASICZBR3", deviceJoinName: "SONOFF Basic (R3 Zigbee)"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "eWeLink", model: "SA-003-Zigbee", deviceJoinName: "eWeLink SmartPlug (SA-003)", ocfDeviceType: "oic.d.smartplug"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "SONOFF", model: "BASICZBR3", deviceJoinName: "SONOFF Basic (R3 Zigbee)", ocfDeviceType: "oic.d.smartplug"
```
Comment (greens) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy:37]: @FenMei007S is there a ticket for these icon changes or are you just correcting them while you're here?

Comment (FenMei007S) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy:37]: https://smartthings.atlassian.net/browse/ICP-10286

File: devicetypes/smartthings/zigbee-switch.src/i18n/messages.properties (hunk 1 above)
Comment (FenMei007S) [reply devicetypes/smartthings/zigbee-switch.src/i18n/messages.properties]: Dear tom, sorry for make confused, it is ICP-10286

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy (hunk 2 above)
Comment (FenMei007S) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy:37]: https://smartthings.atlassian.net/browse/ICP-10286

**Comment by FenMei007S** [comment]
plz review and merge it,we need to start certification for it

**Review by tpmanley — APPROVED** [review]

**Pull request merged**